dev/builddeps-veloxbe.sh

```diff
@@ -103,7 +103,7 @@ if [ "$SKIP_BUILD_EP" != "ON" ]; then
     cd $GLUTEN_DIR/ep/build-velox/src
     ./get_velox.sh --enable_hdfs=$ENABLE_HDFS --build_protobuf=$BUILD_PROTOBUF --enable_s3=$ENABLE_S3
     ./build_velox.sh --enable_s3=$ENABLE_S3 --build_type=$BUILD_TYPE --enable_hdfs=$ENABLE_HDFS \
-                   --enable_ep_cache=$ENABLE_EP_CACHE --build_benchmarks=$BUILD_BENCHMARKS
+                   --enable_ep_cache=$ENABLE_EP_CACHE --build_tests=$BUILD_TESTS
```

Comment:
Why do we remove `build_benchmarks` option here?

Reply:
I find we never use this option, and Gluten seems not need too, `BUILD_TESTS` is enough.